packages/app/src/contacts/components/sync-contacts-modal/sync-contacts-modal.component.tsx

```diff
@@ -86,61 +92,63 @@ const SyncContactsModal: FunctionComponent<Props> = ({
-          ...messages.helpText,
-          values: {
-            link: (...chunks: ReactNode[]) => (
-              <ModalLink onClick={openHelpWindow}>{chunks}</ModalLink>
-            ),
-          },
-        }}
-      />
-      <ButtonsContainer>
-        <ButtonWrapper>
-          <GoogleButton
-            onClick={() => {
-              onGoogleButtonClick()
-            }}
-            type="dark"
-            label={intl.formatMessage(messages.googleButtonText)}
-            data-testid={SyncContactsModalTestIds.GoogleButton}
-            disabled={disabledOtherMethod}
-          />
+      <ModalContent>
+        <Text
+          displayStyle={TextDisplayStyle.Paragraph2}
+          message={messages.text}
+        />
+        <Text
+          displayStyle={TextDisplayStyle.Paragraph2}
+          message={{
+            ...messages.helpText,
+            values: {
+              link: (...chunks: ReactNode[]) => (
+                <ModalLink onClick={openHelpWindow}>{chunks}</ModalLink>
+              ),
+            },
+          }}
+        />
+        <ButtonsContainer>
+          <ButtonWrapper>
+            <GoogleButton
+              onClick={() => {
+                onGoogleButtonClick()
+              }}
+              type="dark"
+              label={intl.formatMessage(messages.googleButtonText)}
+              data-testid={SyncContactsModalTestIds.GoogleButton}
+              disabled={disabledOtherMethod}
+            />
 
-          <SyncButton
-            labelMessage={messages.outlookButtonText}
-            Icon={IconType.Outlook}
-            onClick={onOutlookButtonClick}
-            data-testid={SyncContactsModalTestIds.OutlookButton}
-            disabled={disabledOtherMethod}
-          />
-          {onAppleButtonClick && (
             <SyncButton
-              labelMessage={messages.appleButtonText}
-              Icon={IconType.Apple}
-              onClick={onAppleButtonClick}
+              labelMessage={messages.outlookButtonText}
+              Icon={IconType.Outlook}
+              onClick={onOutlookButtonClick}
+              data-testid={SyncContactsModalTestIds.OutlookButton}
               disabled={disabledOtherMethod}
             />
-          )}
-          <SyncButton
-            labelMessage={messages.manualImportText}
-            Icon={IconType.Upload}
-            onClick={handleManualImportClick}
-            disabled={disabledOtherMethod}
-          />
-          <InputFileSelect
-            ref={fileInputRef}
-            onManualImportClick={onManualImportClick}
-            onCancelManualImportClick={onCancelManualImportClick}
-          />
-        </ButtonWrapper>
-      </ButtonsContainer>
+            {onAppleButtonClick && (
+              <SyncButton
+                labelMessage={messages.appleButtonText}
+                Icon={IconType.Apple}
+                onClick={onAppleButtonClick}
+                disabled={disabledOtherMethod}
+              />
+            )}
```

Comment:
Why here is a check if 'onAppleButtonClick' method is defined but for others is not?